backend/src/net/wifi.rs

```diff
@@ -801,7 +801,6 @@ pub async fn synchronize_wpa_supplicant_conf<P: AsRef<Path>>(
         .arg("up")
         .invoke(ErrorKind::Wifi)
         .await?;
-    Command::new("dhclient").invoke(ErrorKind::Wifi).await?;
```

Comment:
If I remember, this is redundant because the initialization.sh

Reply:
We aren't using dhclient anymore. Relying on systemd-resolved